pkg/apis/planetscale/v2/vitesscluster_types.go

```diff
@@ -107,6 +107,37 @@ type VitessClusterSpec struct {
 	// not have any prefix (just "flag", not "-flag"). To set a boolean flag,
 	// set the string value to either "true" or "false".
 	ExtraVitessFlags map[string]string `json:"extraVitessFlags,omitempty"`
+
+	// TopoReconciliation can be used to enable or disable registration or pruning of various vitess components to and from topo records.
+	TopoReconciliation TopoReconcileConfig `json:"topoReconciliation,omitempty"`
```

Comment:
```suggestion
	TopoReconcile TopoReconcileConfig `json:"topoReconciliation,omitempty"`
```

Reply:
This goes in the opposite direction of @enisoc's suggested change. I'm going to go with his suggestion because the justification seems solid for expanding rather than contracting. Thanks for the feedback, will resolve this once I implement @enisoc's suggestion.